assets/js/atomic/blocks/product-elements/rating/support.js

```diff
@@ -10,5 +10,14 @@ export const supports = {
 			background: false,
 			link: false,
 		},
+		typography: {
+			fontSize: true,
+			__experimentalSkipSerialization: true,
+		},
+		spacing: {
+			__experimentalSkipSerialization: true,
+			margin: true,
+		},
```

Comment:
We should add `__experimentalSkipSerialization` to color prop as well as per [docs](https://developer.wordpress.org/block-editor/explanations/architecture/styles/#:~:text=Styles%20are%20serialized%20to%20the%20outermost%20HTML%20node%20of%20the%20block%2C%20the%20wrapper.). When adding classes/styles manually we should always add skipSerialization.

Would have skipped over it but while at it can we move __experimentalSkipSerialization for spacing at the end so it's consistent?